src/gt4py/cartesian/gtc/debug/debug_codegen.py

```diff
@@ -0,0 +1,355 @@
+# GT4Py - GridTools Framework
+#
+# Copyright (c) 2014-2024, ETH Zurich
+# All rights reserved.
+#
+# Please, refer to the LICENSE file in the root directory.
+# SPDX-License-Identifier: BSD-3-Clause
+
+
+from collections.abc import Generator
+from contextlib import contextmanager
+
+from gt4py import eve
+from gt4py.cartesian import utils
+from gt4py.cartesian.gtc.common import (
```

Comment:
Since this is a new file, let's impose strictly our coding guidelines. Only import modules, see https://google.github.io/styleguide/pyguide.html#22-imports.

Here I would do
```suggestion
from gt4py.cartesian.gtc import common as gtc_common
```
to distinguish from other `common` modules.